src/main/kotlin/no/nav/syfo/behandler/kafka/sykmelding/KafkaBehandlerSykmelding.kt

```diff
@@ -119,12 +119,12 @@ private fun createAndStoreBehandlerFromSykmelding(
         fornavn = sykmeldingBehandler.fornavn,
         mellomnavn = sykmeldingBehandler.mellomnavn,
         etternavn = sykmeldingBehandler.etternavn,
-        herId = sykmeldingBehandler.her?.toInt(),
-        hprId = sykmeldingBehandler.hpr?.toInt(),
-        telefon = sykmeldingBehandler.tlf?.removePrefix("tel:"),
+        herId = sykmeldingBehandler.her?.toIntOrNull(),
```

Comment:
Disse id'ene, er det en avhengighet som krever at disse er Int?

Reply:
Ja, vi forventer at de er Int'er. Det dukket opp sykmeldinger der de var tom streng, så i denne sammenhengen synes jeg det riktigste å likestille det med null.

Reply:
Så de kommer inn som strenger, men vi konverterer til Int?

Reply:
Men risikerer vi ikke å miste data ved en slik konvertering? Hvis det ikke skal gjøres regneoperasjoner på ID, kan den ikke bare likesågodt være String?

Reply:
Vår modell forutsetter at de er int'er (og det er det de skal være). Vi prosesserer sykmeldinger for å finne behandlere vi kan sende dialogmeldinger til - vi kan ikke bruke ugyldige id'er i en sånn sammenheng, så det er ikke noe problem å filtrere bort slike forekomster.